net/mqttcli/Portfile

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem           1.0
+PortGroup            golang 1.0
+
+go.setup             github.com/subpop/mqttcli 0.2.6 v
```

Comment:
We usually also add a `revision` field.

`revision` is used to indicate that a port has changed/upated in a situation where its current version isn't changing.

Reply:
Oh yes. I forgot about that. Thanks for the reminder!